include/openssl/arm_arch.h

```diff
@@ -90,7 +90,15 @@
 #define ARMV8_NEOVERSE_V2 (1 << 14)
 
 // ARMV8_DIT indicates support for the Data-Independent Timing (DIT) flag.
-#define ARMV8_DIT (1 << 14)
+#define ARMV8_DIT (1 << 15)
```

Comment:
this indicates that DIT is supported on the current CPU, right? if so, suggest making the name more descriptive like `ARMV8_DIT_SUPPORTED`